docs/v3/identity.md

```diff
@@ -0,0 +1,261 @@
+# Multi-wallet Identity in V3
+
+## Summary
+
+* XMTP v3 now supports multiple wallets per user, including [smart wallets](https://community.xmtp.org/t/xip-44-smart-contract-wallet-support/627).
+* User wallets now belong to a shared Inbox with a single Inbox ID.
+* The first wallet used to create an Inbox is a *recovery wallet*.
+  * A recovery wallet can remove other wallets.
+  * A user can use the recovery wallet to set a different recovery wallet.
+  * A user with an existing V2 wallet gets an Inbox with that wallet as the recovery wallet.
+* The Inbox can have multiple *associated wallets* (besides the recovery wallet).
+  * Users can send a message to any name belonging to an associated wallet in the Inbox.
```

Comment:
```suggestion
  * Users can send a message to any name(ENS, cb.id, farcaster id, etc.) belonging to an associated wallet in the Inbox.
```
Would it better if we mention popular names here?

Reply:
Fine by me.